lib/ruby_home/config/characteristics.yml

```diff
@@ -30,6 +30,18 @@
     ValidValues:
       '0': Inactive
       '1': Active
+- :name: :active_identifier
+  :description: Active Identifier
+  :uuid: 000000E7-0000-1000-8000-0026BB765291
+  :format: uint32
+  :unit: nil
+  :permissions:
+  - securedRead
```

Comment:
I assumed this should be `securedRead` for all of the characteristics I added in this PR, I didn't see anything that would map to this field.

Reply:
We dont actually use these permissions for anything. I will submit a subsequent PR to remove them. For now they can all be `securedRead` 👍